src/views/Layout.jsx

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
-import { Outlet } from 'react-router-dom';
+import { Outlet, NavLink } from 'react-router-dom';
```

Comment:
Nice work consolidating the `react-router-dom` imports!

Reply:
Same comment as https://github.com/the-collab-lab/tcl-75-smart-shopping-list/pull/36#discussion_r1739938232